### ISSUE TITLE
Add checkbox option for participant country to "All Responses" download

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#800]
# TL;DR

In the "All Responses" study view (https://lookit.mit.edu/exp/studies/###/responses/all/), add a checkbox to the list of characteristics that includes the family's country of residence.

# Narrative
As a researcher, I want to be able to see the location of each participant with their responses so I can pay them in the correct currency

# Acceptance Criteria

- [ ] Ensure country field from User data is appropriate for use as place-of-residence
- [ ] Add country to set of fields available for addition to Response object (if not already)
- [ ] Checkbox added to front end
- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
